### ISSUE TITLE
The modules and filter will not install in Moodle 3.0

### ISSUE THREAD

**Issue by ttsilva90** [kaltura/moodle_plugin#68]
Four of the Kaluta plugins (the three modules and the filter) will not install on a Moodle 3.0 site. Attempting to install them results in the following errors:

Unsupported $module syntax detected in version.php of the mod_kalvidassign plugin.
Plugin mod_kalvidassign does not declare valid $plugin->component in its version.php.
Unsupported $module syntax detected in version.php of the mod_kalvidpres plugin.
Plugin mod_kalvidpres does not declare valid $plugin->component in its version.php.
Unsupported $module syntax detected in version.php of the mod_kalvidres plugin.
Plugin mod_kalvidres does not declare valid $plugin->component in its version.php.
Plugin filter_kaltura does not declare valid $plugin->component in its version.php.

Debug info: Unsupported $module syntax detected in version.php
Error code: detectedbrokenplugin

**Comment by kaltura-hooks** [comment]
Hi @ttsilva90,

Thank for you reporting an issue and helping improve Kaltura!

To get the fastest response time, and help the maintainers review and test your reported issues or suggestions, please ensure that your issue includes the following (please comment with more info if you have not included all this info in your original issue):
- Is the issue you're experiencing consistent and across platforms? or does it only happens on certain conditions?
  please provide as much details as possible.
- Which Kaltura deployment you're using: Kaltura SaaS, or self-hosted?
  If self hosted, are you using the RPM or deb install?
- Packages installed.
  When using RPM, paste the output for:

```
    # rpm -qa "kaltura*"
```

```
For deb based systems:
```

```
    # dpkg -l "kaltura-*"
```
- If running a self hosted ENV - provide the MySQL server version used
- If running a self hosted ENV - is this a single all in 1 server or a cluster?
- If running a self hosted ENV, while making the problematic request, run:

```
    # tail -f /opt/kaltura/log/*.log /opt/kaltura/log/batch/*.log | grep -A 1 -B 1 --color "ERR:\|PHP\|trace\|CRIT\|\[error\]"
```

   and paste the output.
- When relevant, provide any screenshots or screen recordings showing the issue you're experiencing.

For general troubleshooting see:
https://github.com/kaltura/platform-install-packages/blob/Jupiter-10.13.0/doc/kaltura-packages-faq.md#troubleshooting-help

If you only have a general question rather than a bug report, please close this issue and post at:
http://forum.kaltura.org

Thank you in advance,

**Comment by muli** [comment]
Hi @ttsilva90,

Thanks for reporting this issue! It looks like Moodle dropped supported for `$module->xxx` syntax and replaced it with `$plugin->xxx` in version.php files. Please see https://tracker.moodle.org/browse/MDL-43040 for more details.

Please note that our plugin was not tested & certified against Moodle 3.0 yet (we officially support Moodle 2.9 and below), but rest assured that this issue will be fixed when we release our Moodle 3.0 certified plugin version.

Again, thanks for bringing this to our attention!

**Comment by muli** [comment]
This issue is fixed in version 4.0.08. Thanks for your report!
